translations/pl.js

```diff
@@ -0,0 +1,121 @@
+    "Conditional Start Event (non-interrupting)": "Warunkowe zdarzenie początkowe (nieprzerywalne)",
+    "Connect using Association": "Połącz z użyciem asocjacji",
+    "Connect using DataInputAssociation": "Połącz z użyciem DataInputAssociation",
+    "Connect using Sequence/MessageFlow or Association": "Połącz z użyciem sekwencji/przepływu wiadomości lub asocjacji",
+    "Create DataObjectReference": "Utwórz referencję obiektu danych",
+    "Create DataStoreReference": "Utwórz referencję magazynu danych",
+    "Create EndEvent": "Utwórz zdarzenie końcowe",
+    "Create Gateway": "Utwórz bramę",
+    "Create Group": "Utwórz grupę",
+    "Create Intermediate/Boundary Event": "Utwórz pośrednie/graniczne zdarzenie",
+    "Create Pool/Participant": "Utwórz pulę/uczestnika",
+    "Create StartEvent": "Utwórz zdarzenie początkowe",
+    "Create Task": "Utwórz zadanie",
+    "Create expanded SubProcess": "Utwórz rozszerzony podproces",
+    "Data Object Reference": "Referencja obiektu danych",
+    "Data Store Reference": "Referencja magazynu danych",
+    "Default Flow": "Domyślny przepływ",
+    "Divide into three Lanes": "Podziel na trzy pasy",
+    "Divide into two Lanes": "Podziel na dwa pasy",
+    "Empty Pool": "Pusta pula",
+    "Empty Pool (removes content)": "Pusta pula (usuwa zawartość)",
+    "End Event": "Zdarzenie końcowe",
+    "Error Boundary Event": "Graniczne zdarzenie błędu",
+    "Error End Event": "Końcowe zdarzenie błędu",
+    "Error Start Event": "Początkowe zdarzenie błędu",
+    "Escalation Boundary Event": "Graniczne zdarzenie eskalacji",
+    "Escalation Boundary Event (non-interrupting)": "Graniczne zdarzenie eskalacji (nieprzerywalne)",
+    "Escalation End Event": "Końcowe zdarzenie eskalacji",
+    "Escalation Intermediate Throw Event": "Pośrednie zdarzenie rzucające eskalacji",
+    "Escalation Start Event": "Początkowe zdarzenie eskalacji",
+    "Escalation Start Event (non-interrupting)": "Początkowe zdarzenie eskalacji (nieprzerywalne)",
+    "Event Sub Process": "Zdarzenie podprocesu",
+    "Event based Gateway": "Brama zdarzeniowa",
+    "Exclusive Gateway": "Brama alternatywa wykluczająca",
+    "Inclusive Gateway": "Brama alternatywa",
+    "Intermediate Throw Event": "Pośrednie zdarzenie rzucające",
+    "Link Intermediate Catch Event": "Łączące pośrednie zdarzenie łapiące",
+    "Link Intermediate Throw Event": "Łączące pośrednie zdarzenie rzucające",
+    "Loop": "Pętla",
+    "Manual Task": "Zadanie ręczne",
+    "Message Boundary Event": "Zdarzenie wiadomości granicznej",
+    "Message Boundary Event (non-interrupting)": "Zdarzenie wiadomości granicznej (nieprzerywalne)",
+    "Message End Event": "Zdarzenie wiadomości końcowej",
+    "Message Intermediate Catch Event": "Zdarzenie łapiące wiadomości pośredniej",
+    "Message Intermediate Throw Event": "Zdarzenie rzucające wiadomości pośredniej",
+    "Message Start Event": "Zdarzenie wiadomości początkowej",
```

Comment:
```suggestion
    "Message End Event": "Zdarzenie końcowe wiadomości",
    "Message Intermediate Catch Event": "Zdarzenie łapiące wiadomości",
    "Message Intermediate Throw Event": "Zdarzenie rzucające wiadomości",
    "Message Start Event": "Zdarzenie początkowe wiadomości",
```

Dostosuj również pozostałe podtypy zdarzeń. To nie wiadomość, tylko zdarzenie jest początkowe/pośrednie/kończące.

EN: Adjust also the remaining specialised events. It's not the message but the event which is described with "start/intermediate/end".